packages/contracts-bedrock/snapshots/storageLayout/OptimismPortal2.json

```diff
@@ -132,11 +132,18 @@
     "slot": "62",
     "type": "contract IAnchorStateRegistry"
   },
+  {
+    "bytes": "20",
+    "label": "ethLockbox",
+    "offset": 0,
+    "slot": "63",
+    "type": "contract IETHLockbox"
+  },
   {
     "bytes": "1",
     "label": "superRootsActive",
     "offset": 20,
-    "slot": "62",
+    "slot": "63",
```

Comment:
Slot difference is OK because this variable has not yet been shipped to production